### DISCUSSION TITLE
what is the ip_adapter_image optional argument in IP adapter pipeline?

### DISCUSSION THREAD

**Discussion by nivedwho** [huggingface/diffusers#6735] [General]
In the T2I adapter pipeline I saw [this](https://github.com/huggingface/diffusers/blob/8581d9bce43ac2747199f848a9ac861352495d10/src/diffusers/pipelines/t2i_adapter/pipeline_stable_diffusion_xl_adapter.py#L932) additional image encoding was being passed to the unet, did not quite find anything related in the original paper. Can someone help me understand what this does exactly?

**Comment by fabiorigano** [comment]
Hi @nivedwho, ip_adapter_image is an optional image input used to condition the unet, see https://huggingface.co/docs/diffusers/main/en/api/loaders/ip_adapter as reference
